.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: debian:latest
+    steps:
+      - run:
+          name: Install required packages
+          command: |
+            apt-get update -y
+            apt-get install -y curl git make python-setuptools tar
+      - run:
+          name: Install Arduino platform
```

Comment:
Why are we installing them from here rather than `apt-get`?

Reply:
See commit message; the version of arduino packaged in debian (1.0.5) is too old. Specifically, we require the `(double)` overload of the `String` constructor, which doesn't seem to be present in 1.0.5.

Reply:
Even on the `debian:latest` image, urgh, wow. Makes sense then 👍